docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,182 @@ eclagent:
+
+    <para>The LDAP administrators account must have administrator rights to
+    all of the Base DNs used by the HPCC Systems platform. In a cloud
+    deployment these credentials can be exposed. A good practice then, for
+    these administrator credentials is to be secured either using Kubernetes
+    secrets or the Hashicorp Vault.</para>
+
+    <sect2 id="SecuringCredentials_For_KubernetesSecrets">
+      <title>Securing Credentials in Kubernetes</title>
+
+      <para>To create a secret in Kubernetes to store the LDAP administrators
+      user account credentials, use a command line interface to Kubernetes,
+      and execute a command similar to the following example:</para>
+
+      <programlisting lang="bash">kubectl create secret generic admincredssecretname --from-literal=username=hpcc_admin \
+  --from-literal=password=t0pS3cr3tP@ssw0rd
+</programlisting>
+
+      <para>In the above example, the Kubernetes secret name, is
+      "admincredssecretname" and it contains the LDAP administrators account
+      "username" and "password" key/values. This stores the LDAP
+      administrators username and password as a Kubernetes secret.. Any
+      additional properties are ignored.</para>
+
+      <para>You can verify the secret you just created by executing the
+      following Kubernetes command.</para>
+
+      <programlisting lang="bash">kubectl get secret admincredssecretname</programlisting>
+
+      <para>For more information about Kubernetes see the appropriate
+      Kubernetes documentation for your implementation.</para>
+
+      <sect3 id="Usiing_TheKubernetesSecret">
+        <title>Using the Kubernetes Secret</title>
+
+        <para>To deploy the Kubernetes secrets override the "secrets:" section
+        in HPCC-Platform/helm/hpcc/values.yaml, or deploy with your own
+        customized chart. For more information about customizing your HPCC
+        Systems containerized deployment see the above sections on
+        customization techniques.</para>
+
+        <para>In your chart, create a unique key name used to reference the
+        secret, and set it to the name of secret that you created in the
+        previous step. In the above example it was
+        "admincredssecretname".</para>
+
+        <para>You can optionally define additional secrets as required by your
+        platform security configuration. Each of these secrets would be
+        created as described above and given unique names. The example below
+        indicates how you can add any additional credentials or secrets to
+        your Helm chart(s) if necessary.</para>
+
+        <para>The "admincredsmountname" key/value pair already exists by
+        default in the delivered HPCC Systems values.yaml file. The key is
+        referenced in the component's ldap.yaml file. You may override these
+        and add additional key/values as needed. The following example
+        illustrates adding "addiotionalsecretname" and that name must match
```

Comment:
Is **addiotionalsecretname** correct? Seems like it should be **additionalsecretname**

Reply:
good catch!